### ISSUE TITLE
Add files to work item

### ISSUE THREAD

**Issue by AmitShadadi** [kemmis/Kemmis.MyWorkItemsOnPendingChangesPage#14]
Hi, i like your work, would be awesome if we could separate the check in file list to groups, i mean associate files to work item and exclude or include check in by work item and not file by file - that will make so many programmers's life much easier, in my project i work on many tasks and bugs simultaneously and when the big day has come when i have to do the check in, it takes ages to pick the files you need for the specific check in , 

Thanks!

**Comment by eFloh** [comment]
tip: use separate workspaces for different Tasks, have no hassle...

**Comment by bartjanson** [comment]
indeed, make sure you use git, separate branch per task / bug, problem solved.
